include/openthread/backbone_router_ftd.h

```diff
@@ -127,7 +141,10 @@ void otBackboneRouterGetConfig(otInstance *aInstance, otBackboneRouterConfig *aC
 otError otBackboneRouterSetConfig(otInstance *aInstance, const otBackboneRouterConfig *aConfig);
 
 /**
- * This function explicitly registers local Backbone Router configuration.
+ * Explicitly registers local Backbone Router configuration. A Server Data Request message
```

Comment:
Split brief and detail into separate blocks.

Reply:
Thanks, done.